spinalcordtoolbox/deepseg/inference.py

```diff
@@ -125,6 +125,10 @@ def segment_non_ivadomed(path_model, model_type, input_filenames, threshold, use
         fnames_out, targets = inference(path_img=fname_in, tmpdir=tmpdir, predictor=net, device=device)
         for fname_out, target in zip(fnames_out, targets):
             im_out = Image(fname_out)
+            # Apply postprocessing (replicates existing functionality from ivadomed package)
+            # 1. Keep the largest connected object
+            im_out.data = keep_largest_object(im_out.data)
```

Comment:
> But, since this is applied by default in segment_non_ivadomed function, it might also be applied in the case of lesion segmentation models (we have to be careful about this).

I had the exact same thought, especially in the context of lesion segmentation. So, I was presuming that we would make this behavior opt-in, and re-use the existing `-remove-small` argument that already exists in `sct_deepseg`.

I'm wondering, since `-remove-small` is just an option to turn on ivadomed postprocessing, and `keep_largest_object` is based on the ivadomed code... were they both based on the same ivadomed code? Can we re-use the existing ivadomed code, and thus be able to re-use `-remove-small` too?

Reply:
Thanks for the suggestion! One of the reasons I didn't call the ivadomed function directly is that I/we don't know if ivadomed will be supported in the future! There are active projects that are trying to port ivadomed models into monai so I wasn't sure if importing the postprocessing functions from ivadomed would be future compatible? what do you think?

Reply:
Ah, sorry! I mean like, copying the function entirely? (If we're going to make a new function inside of `math.py` anyway, we might as well make it identical to the existing logic?)

Reply:
Oh! My mistake. It looks like the function you added is identical to the [ivadomed function](https://github.com/ivadomed/ivadomed/blob/1fccf77239985fc3be99161f9eb18c9470d65206/ivadomed/postprocessing.py#L99-L116) already!

In that case, all we have to do is tap into the value of `sct_deepseg -largest` to control this option.